MeshGeoToolsLib/BoundaryElementsSearcher.cpp

```diff
@@ -58,6 +58,28 @@ std::vector<MeshLib::Element*> const& BoundaryElementsSearcher::getBoundaryEleme
     }
 }
 
+std::vector<std::pair<std::size_t, unsigned>> const&
+BoundaryElementsSearcher::getBulkIDs(GeoLib::GeoObject const& geometry)
+{
+    switch (geometry.getGeoType()) {
+    case GeoLib::GEOTYPE::POINT:
+        return getBulkIDsAtPoint(
+            *dynamic_cast<const GeoLib::Point*>(&geometry));
+        break;
+    case GeoLib::GEOTYPE::POLYLINE:
+        return getBulkIDsAlongPolyline(
+            *dynamic_cast<const GeoLib::Polyline*>(&geometry));
+        break;
+    case GeoLib::GEOTYPE::SURFACE:
+        return getBulkIDsOnSurface(
+            *dynamic_cast<const GeoLib::Surface*>(&geometry));
+        break;
+    default:
+        const static std::vector<std::pair<std::size_t, unsigned>> dummy;
+        return dummy;
```

Comment:
Shouldn't that case rather be an error?

Reply:
In principle you are right. I made it consistent with the existing code. The new method `getBulkIDs()` has the same behaviour as the similar method `getBoundaryElements()`. I don't want to break the consistency in this PR.  @chleh Is this okay for you?

Reply:
OK.